scripts/generate-types.js

```diff
@@ -42,24 +51,67 @@ const getHtmlAttributeInterfaces = (typesFileContent, className, properties) =>
 	typesFileContent += `\tclass?: string;${EOL}`;
 	properties.forEach(property => {
 		// HTML attributes can be one of two types: boolean or string
-		const htmlType = property.Type.toLowerCase() === 'boolean' ? property.Type : 'string';
-		typesFileContent += generatePropertyLine(property.Attribute, htmlType, property.Description);
+		// const htmlType = property.Type.toLowerCase() === 'boolean' ? property.Type : 'string';
+		const tsType = jsTypeToTypescriptType(property.Type);
+		typesFileContent += generatePropertyLine(property.Attribute, tsType, property.Description);
```

Comment:
Please, revert these changes. According to HTML spec, the accepted types for html attributes are string or boolean.
Actually, you can test yourself with a date-picker's function `isDisabledDate`, this code will not trigger the function:
```ts
  const func = () => {
    console.log('called!');
  }

<ts-date-picker isDisabledDate={func}></ts-date-picker>
```

I think the correct way is to do it via ref:
```ts
import React, {useEffect, useRef} from "react";
import "@tradeshift/elements.date-picker";
import {TSDatePicker} from "@tradeshift/elements.date-picker";

function DatePicker() {
  const dateRef = useRef<TSDatePicker & HTMLElement>(null);
  const func = () => {
    console.log('called!');
  }
  useEffect(() => {
    if (dateRef && dateRef.current) {
      dateRef.current.isDisabledDate = func;
    }
  }, [dateRef]);
  return <ts-date-picker ref={dateRef}></ts-date-picker>;
}
```

Reply:
Good catch!

Updated.
I changed 'boolean' to be of a similar type as React does for HTML attributes

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L54